components/shared/index.js

```diff
@@ -5,13 +5,14 @@ import icons from './icons';
 import masks from './masks';
 import * as theme from './theme';
 import BREAKPOINTS from './breakpoints';
-import uniqId from './uniqId';
 import * as INPUT_STYLE from './inputStyle';
 import hexToRgba from './hexToRgba';
 import shadow from './shadow';
 import HiddenInput from './HiddenInput';
 import normalizeChars from './normalizeChars';
 
+export * from './uniqId';
+
```

Comment:
This is a common way to export modules using named exports. It doesn't really improve performance, but it keeps the code simpler. When we use `export * from "./module"`, we make the module responsible for exporting what is needed. If a function has the export, it's already visible to everything.

As for why we use named exports, at the very least it's considered good practice. Named exports offer more guarantees - if a function's name changes, for example, it will require every other place to update the name, making it more consistent.